ifbcat_api/management/commands/load_keywords_translate_from_csv.py

```diff
@@ -16,33 +16,30 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         if not os.path.exists(options["file"]):
```

Comment:
If the file is absent it have to be created, here is will stop which is not the expected behavior.

Reply:
> If the file is absent it have to be created, here is will stop which is not the expected behavior.

We need to have the default file with the keywords translated (En & Fr).
If we create a file because the default path doesnt exist, how could we write to the new file the keywords for translation macthing .
> I thought the purpose was to translate the keywords in DB (Fr to En)

Reply:
The purpose is indeed to translate the keywords in DB (Fr to En).

If the file is absent then it will be created, and later we open it, translate it and re-run the commande to apply the new translation that were made into the file.

Also creating the file when missing allows to verify that the command is working properly to add new keywords to translate